_component_design/footer.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Footer
+parent: Navigation
+layout: component
+category: Components
+usage: >
+The footer lives at the bottom of the page to help visitors find
```

Comment:
Sorry - this is a hard break, I know. Please review the rest and I will update this accordingly.